### PR TITLE
Format markdown in description

### PR DESCRIPTION
**Pull request by vvuunnqq**
+ Fix: format markdown in description
+ Fix: infinite scroll too early

### CODE REVIEW

**Comment by vvuunnqq** [comment]
result on my local machine
image

File: lib/companies_web/views/company_view.ex [hunk 1]
```diff
@@ -23,7 +23,8 @@ defmodule CompaniesWeb.CompanyView do
   end
 
   def markdown_format(markdown) do
-    markdown
+    Earmark.as_html!(markdown)
+    |> Phoenix.HTML.raw()
```
Comment (doomspork) [inline lib/companies_web/views/company_view.ex:27]: I've been trying to keep the codebase as closely aligned to the community styleguide as possible: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines

```suggestion
    markdown
    |> Earmark.as_html!()
    |> Phoenix.HTML.raw()
```

**Review by doomspork — CHANGES_REQUESTED** [review]
Thank you again @vvuunnqq! 🎉 Small piece of feedback to keep our code consistent with the styleguide I try to align with (and configure Credo for).

**Comment by vvuunnqq** [comment]
Thanks, I updated

**Review by doomspork — APPROVED** [review]
You're the best @vvuunnqq! 🚀

**Pull request merged**